pmpro-sponsored-members.php

```diff
@@ -1848,21 +1962,13 @@ function pmprosm_the_content_account_page($content)
                             <li><?php echo $code_url['name'];?>: <strong><a target="_blank" href="<?php echo $code_url['url'];?>"><?php echo $code_url['url'];?></a></strong></li>
                         <?php } ?>
                     </ul>
-                <?php } // hide_display_discount_code ?>
+                <?php } // hide_display_discount_code 
 
-                <div class="pmpro_message pmpro_default">
-					<?php if(empty($code->uses)) { ?>
-						<?php _e("This code has unlimited uses.", "pmpro_sponsored_members");?>
-					<?php } else { ?>
-						<?php printf(__("%s/%s uses.", "pmpro_sponsored_members"), count($member_ids), $code->uses);?>
-					<?php } ?>
-				</div>
-				<?php
                     // use same account display as in admin
-                        if ( ! empty( $member_ids ) ) {
+                        //if ( ! empty( $member_ids ) ) {

```

Comment:
can just remove these instead of commenting them out if they aren't needed anymore.